CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ Install luajit (`brew install luajit`). Run `make osx`.
 
 ### Windows 32 and 64
 
```

Comment:
And I thought it was wild when Microsoft jumped from Windows 8 to Windows 10. Skipping ahead some 18000 years is something else...